Build/AdditionalProjectTransforms.xslt

```diff
@@ -0,0 +1,5 @@
+<xsl:if test="/Input/Properties/DefineNET4DetectionConstant = 'True'">
+  <PropertyGroup>
+    <DefineConstants Condition=" '$(TargetFrameworkVersion)' == 'v4.0' ">NET4</DefineConstants>
```

Comment:
I've updated the PR but it fails loading the project on Xamarin platforms. It looks like it cannot calls ```System.Version.Parse()``` and/or ```System.Version.CompareTo()``` within the csproj.
I believe it has something to do with XBuild not supporting this (or did we switch to msbuild and it's something else?).